spanner/tests/system/test_system.py

```diff
@@ -282,6 +282,20 @@ def test_create_database(self):
             for database in Config.INSTANCE.list_databases()]
         self.assertIn(temp_db_id, database_ids)
 
+    def test_table_not_found(self):
```

Comment:
It seems that the first DDL ("CREATE TABLE") will succeed, so this test should have a "clean up" that deletes the table.

Reply:
![screenshot from 2017-12-20 09-18-20](https://user-images.githubusercontent.com/520669/34219649-c0fa0244-e566-11e7-8092-57bd798559fb.png)

I'm going to send a commit that does this cleanup ASAP.